src/main/java/seedu/address/model/person/Person.java

```diff
@@ -56,6 +56,10 @@ public Person(Name name, Phone phone, Email email, Address address, Optional<Rol
 
     public void setOwnWedding(Wedding wedding) {
         ownWedding = wedding;
```

Comment:
i think the check should come before the assignment

Reply:
fixed